src/ScheduleFunctions.cpp

```diff
@@ -1193,10 +1193,17 @@ class InjectFunctionRealization : public IRMutator {
             internal_assert(iter != dims.end());
             start_fuse = (int)(iter - dims.begin());
         }
+
+        const auto &env_iter = env.find(fuse_level.func());
+        internal_assert(env_iter != env.end());
+        const auto &parent_func = env_iter->second;
+
+        const vector<Dim> &parent_dims = (fuse_level.stage_index() == 0) ? parent_func.definition().schedule().dims() : parent_func.update(fuse_level.stage_index() - 1).schedule().dims();
```

Comment:
style nit: long line should be broken, eg

```
const vector<Dim> &parent_dims = (fuse_level.stage_index() == 0) 
    ? parent_func.definition().schedule().dims() 
    : parent_func.update(fuse_level.stage_index() - 1).schedule().dims();
```

Reply:
Done, I was hoping that 'make format' would fix it, but apparently not.

Reply:
Ugh, now clang-format pre-check complains, is there a way to suppress it?

Reply:
I would trying adding parenthesis around the entire expression and then just accept whatever 'make format' decides to do with it.

Reply:
Thanks, 'make format' would just make it one line no matter what I do. I've split it into two lines, one of them is still pretty long, but maybe closer to acceptable.